ewebsock/src/lib.rs

```diff
@@ -133,12 +133,24 @@ pub struct Options {
     ///
     /// Ignored on Web.
     pub max_incoming_frame_size: usize,
+
+    /// Additional Request headers
+    pub additional_headers: Option<Vec<(String, String)>>,
```

Comment:
What is the difference between `None` and `Some(vec![])`?
If there is none, just remove the `Option`